bindings/matrix-sdk-ffi/src/api.udl

```diff
@@ -152,10 +152,10 @@ interface Client {
     void login(string username, string password, string? initial_device_name, string? device_id);
 
     [Throws=ClientError]
-    void restore_login(string restore_token);
+    void restore_login(Session session);
```

Comment:
Should we maybe call this `restore_session` now?

Reply:
It's called `restore_login` in the main crate, so if we change it we should change both. Can be a follow-up PR if we decide to do it.